src/bun.js/api/bun/subprocess.zig

```diff
@@ -376,12 +376,18 @@ const Readable = union(enum) {
     inherit: void,
     ignore: void,
     closed: void,
-    buffer: []u8,
+    /// Eventually we will implement Readables created from blobs and array buffers.
+    /// When we do that, `buffer` will be borrowed from those objects.
+    ///
+    /// When a buffered `pipe` finishes reading from its file descriptor,
+    /// the owning `Readable` will be convered into this variant and the pipe's
+    /// buffer will be taken as an owned `CowString`.
+    buffer: CowString,
```

Comment:
this string is always owned? if so, then `CowString` is the wrong abstraction for this field

Reply:
It is for now, but it won't be in the future.